spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc

```diff
@@ -419,6 +423,11 @@ prefix::
   A prefix to be added to the name of the `destination` exchange.
 +
 Default: "".
+queueNameIsGroupOnly::
+  When true, the consumer consumes from a queue with a name equal to the `group`; otherwise the queue name is `destination.group`.
```

Comment:
The "consumer consumes" is confusing somehow on the producer side.
Although I would say the `queue` per se is confusing here anyway...